src/domains/ecommerce-app/06_functions.tf

```diff
@@ -0,0 +1,128 @@
+resource "azurerm_resource_group" "ecommerce_functions_rg" {
+  name     = format("%s-fn-rg", local.project)
+  location = var.location
+
+  tags = var.tags
+}
+
+# Subnet to host ecommerce reporting function
+module "ecommerce_reporting_functions_snet" {
+  source                                         = "git::https://github.com/pagopa/azurerm.git//subnet?ref=v4.3.2"
+  name                                           = "${local.project}-reporting-fn-snet"
+  address_prefixes                               = [var.cidr_subnet_ecommerce_functions]
+  resource_group_name                            = local.vnet_resource_group_name
+  virtual_network_name                           = data.azurerm_virtual_network.vnet.name
+  enforce_private_link_endpoint_network_policies = true
+
+  service_endpoints = [
+    "Microsoft.Web",
```

Comment:
please add `Microsoft.Storage` as service endpoint for the storage of the azure function

Reply:
fixed in https://github.com/pagopa/pagopa-infra/commit/a699345aa42f99288e104bc3e7d9ed2039657769